news/2022-09-01-how-zetera-became-a-medal-hunter.md

```diff
@@ -0,0 +1,79 @@
+
+![](/wiki/shared/news/2022-09-30-how-zetera-became-a-medal-hunter/oldmedals.jpg)
+
+Every time new ones were added, he'd try and find the meaning behind them and eventually found himself building a habit out of it. Discord and the likes did not exist back in those days, so a lot of time was spent hanging out on the forums, `#osu` and in-game DMs, with lots of players often coming together and collaborating to work out the solutions together.
+
+In October of 2017, a new [Halloween-themed update](https://osu.ppy.sh/home/news/2017-10-31-halloween-descends) was released for osu!, bringing with it a total of 9 new Hush-Hush medals. It had been a year since the last batch of Hush-Hush medals at that point and finally, the community had more cryptic puzzles (in the form of medal flair texts and icons) to chew on.
+
+Since getting batches of medals was still new, there wasn't a lot of knowledge behind what made the medal system tick, especially since the whole thing was rewritten back in 2016. Nowadays, the community knows exactly what to look for and how extensive the system is, but back then the community's ideas were much more abstract than what was actually possible, simply because it was really hard to gauge just how precise requirements were for any given medal.
+
+Figuring out how to solve a medal is often a complicated process, as the only real hints provided are the medal's name, icon and some mouse-over text known internally as a 'flair'. At times, even these are not enough, and some medals have even had all three of these things changed over the years.
+
+Despite these challenges, the community managed to figure out the solutions to all of 2017's new medals save for one: "The Girl in the Forest". The community first deduced that the relevant map was [Takuya](https://osu.ppy.sh/users/342372)'s set of [Pika Girl](https://osu.ppy.sh/beatmapsets/40440#osu/128718) by [S3RL](https://osu.ppy.sh/beatmaps/artists/9) after linking the hints and flair text back to it. Zetera then immediately went and SS'd the map, but that didn't work. Maybe a mod was required, or the map needed to be FC'd in all game modes consecutively to get the medal, but that didn't work either. Maybe it had to be an SS on all game modes?
+
+The limits of the medal system needed to be examined more closely, albeit with educated guesswork and deduction. Surely it can't check when you lost your full combo or when an SS was lost, so the only things left would be the miss count, max combo, counts of 300s, 100s, 50s and misses, playing in different modes and a couple of other things, such as at what time was the score submitted. At the time, the flair text read as "First Generation in more than one ways", which lead Zetera to think that it might be related to Pokémon or perhaps the first generation of influential mappers, but in the end, found himself stumped.
```

Comment:
```suggestion
The limits of the medal system needed to be examined more closely, albeit with educated guesswork and deduction. Surely it can't check when you lost your full combo or when an SS was lost, so the only things left would be the miss count, max combo, counts of 300s, 100s, 50s and misses, playing in different modes and a couple of other things, such as at what time was the score submitted. At the time, the flair text read as "First Generation in more than one ways", which lead Zetera to think that it might be related to Pokémon or perhaps the first generation of influential mappers, but in the end, he found himself stumped.
```